packages/core/core/src/Transformation.js

```diff
@@ -601,147 +601,164 @@ export default class Transformation {
 
     return this.loadConfig(configRequest);
   }
-}
 
-type Pipeline = {|
-  id: string,
-  transformers: Array<TransformerWithNameAndConfig>,
-  configs: ConfigMap,
-  options: ParcelOptions,
-  pluginOptions: PluginOptions,
-  resolverRunner: ResolverRunner,
-  workerApi: WorkerApi,
-  postProcess?: PostProcessFunc,
-  generate?: GenerateFunc,
-|};
+  async runTransformer(
+    pipeline: Pipeline,
+    asset: UncommittedAsset,
+    transformer: Transformer,
+    transformerName: string,
+    preloadedConfig: ?Config,
+    configKeyPath: string,
+    parcelConfig: ParcelConfig,
+  ): Promise<Array<TransformerResult>> {
+    const logger = new PluginLogger({origin: transformerName});
+
+    const resolve = async (from: FilePath, to: string): Promise<FilePath> => {
+      let result = nullthrows(
+        await pipeline.resolverRunner.resolve(
+          createDependency({
+            env: asset.value.env,
+            moduleSpecifier: to,
+            sourcePath: from,
+          }),
+        ),
+      );
 
-type TransformerWithNameAndConfig = {|
-  name: PackageName,
-  plugin: Transformer,
-  config: ?Config,
-  configKeyPath: string,
-|};
+      if (result.invalidateOnFileCreate) {
+        // TODO ???
```

Comment:
Not sure what to do in this case... We could send these back to the graph, but somehow they need to also invalidate the cache entry. 😞